src/platform/forms-system/src/js/actions.js

```diff
@@ -1,6 +1,7 @@
 import Raven from 'raven-js';
 import moment from 'moment';
-import { transformForSubmit, recordEvent } from './helpers';
+import { transformForSubmit } from './helpers';
+import recordEvent from 'platform/monitoring/record-event';
```

Comment:
just to help other reviewers, there's a ticket to eventually remove the `recordEvent` helper that lives in forms-system: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/18738

Reply:
Not an issue.
18738 will actually remove the OTHER definition (/platform/forms-system/...), preserving the one that I imported here ('platform/monitoring/record-event'.